charts/kyverno/values.yaml

```diff
@@ -48,23 +48,29 @@ podSecurityContext: {}
 # Optional priority class to be used for kyverno pods
 priorityClassName: ""
 
-antiAffinity:
-  # This can be disabled in a 1 node cluster.
-  enable: true
-
-affinity:
-  podAntiAffinity:
-    preferredDuringSchedulingIgnoredDuringExecution:
+podAntiAffinity:
+  preferredDuringSchedulingIgnoredDuringExecution:
     - weight: 1
       podAffinityTerm:
         labelSelector:
           matchExpressions:
-          - key: app.kubernetes.io/name
-            operator: In
-            values:
-            - kyverno
+            - key: app.kubernetes.io/name
+              operator: In
+              values:
+                - kyverno
         topologyKey: kubernetes.io/hostname
 
+otherAffinity: {}
+  # This can be e.g. nodeAffinity
+
+affinity:
+  enable: true
```

Comment:
I'd recommend simplifying this a bit and remove the `affinity.enable` and just have `affinity.podAntiAffinity.enable` and `affinity.otherAffinity.enable`.  This way there are fewer things someone has to adjust to change affinity behavior.

Reply:
done